src/components/anchor-navigation/anchor-section-divider.d.ts

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+export interface AnchorSectionDividerProps {
+  /** Allows to override existing component styles */
+  styleOverride?: () => object | object;
```

Comment:
same as previous comment, func | object maybe?

Reply:
Same as above<
`styleOverride?: () => object | object` means that it is either a function returning an object, or just an object